app/client.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Navbar from './user/components/Navbar';
-import PostView from './user/post/PostView';
+import { BrowserRouter } from 'react-router-dom';
+import App from './auth/components/App';
 
 require('./globals.scss');
 
 ReactDOM.render(
-  <div className="root">
-    <Navbar />
-    <PostView />
-  </div>,
-  document.getElementById('app'),
+  (
+    <BrowserRouter>
+      <App />
```

Comment:
Again look at this [example](https://github.com/kilkelly/react-passport-redux-example/blob/master/src/client/client.js) and I think a configuration like this (I know the example does not use v4) will be useful in the future when we will be using `redux`. Again these are the norms when we are using `redux`, so better stick to it.

Reply:
The [documentation](http://redux.js.org/docs/advanced/UsageWithReactRouter.html) of React Router for integrating with Redux does not mention this norm. Also the Redux [documentation](http://redux.js.org/docs/advanced/UsageWithReactRouter.html) for integrating with React Router has used a single `App` component.

Reply:
Okay I agree that this is doing the same thing. You are just doing another abstraction on the `routes` which I don't think is any problem. But please keep all the `routes` together the way the documentation has kept in `Root`.